coordinator/src/scheduler.rs

```diff
@@ -133,17 +152,37 @@ fn build_rollover_notification_job(
         // calculates the expiry of the next rollover window. positions which have an
         // expiry before that haven't rolled over yet, and need to be reminded.
         let expiry = coordinator_commons::calculate_next_expiry(OffsetDateTime::now_utc(), network);
-        match get_all_open_positions_with_expiry_before(&mut conn, expiry) {
-            Ok(positions_with_token) => Box::pin({
+        match Position::get_all_open_positions_with_expiry_before(&mut conn, expiry) {
+            Ok(positions) => Box::pin({
                 let notification = notification.clone();
+                let node = node.clone();
                 async move {
-                    for (position, fcm_token) in positions_with_token {
-                        tracing::debug!(trader_id=%position.trader, "Sending reminder to rollover position.");
-                        if let Err(e) = notification_sender
-                            .send(Notification::new(fcm_token.clone(), notification.clone()))
-                            .await
+                    for position in positions {
+                        let trader_id = position.trader;
+                        tracing::debug!(%trader_id, "Sending reminder to rollover position.");
+
+                        let signed_channel = match node
+                            .inner
+                            .get_signed_channel_by_trader_id(trader_id)
                         {
-                            tracing::error!("Failed to send {notification:?} notification: {e:?}");
+                            Err(e) => {
+                                tracing::error!(%trader_id, "Couldn't find signed channel. Skipping proposing rollover. Error: {e:#}");
+                                continue;
```

Comment:
🙃 I think it would be nicer to get rid of the explicit `continue` by defining a helper function to process each `position`. That way we can use `?` and return a `Result` in the helper and simply log for all returned errors in the `for` loop.